tests/process.rs

```diff
@@ -23,6 +23,57 @@ fn test_process() {
         .any(|p| !p.exe().to_str().unwrap_or("").is_empty()));
 }
 
+#[test]
+#[cfg(windows)]
+fn test_process_cwd() {
+    let p = std::process::Command::new("timeout.exe")
+        .arg("/t")
+        .arg("10")
+        .spawn()
+        .unwrap();
+    let pid = p.id() as sysinfo::Pid;
+
+    let mut s = sysinfo::System::new();
+    s.refresh_processes();
+
+    let processes = s.processes();
+    let p = processes.get(&pid);
+
+    if let Some(p) = p {
+        assert_eq!(p.pid(), pid);
+        assert_eq!(p.cwd(), std::env::current_dir().unwrap());
+    } else {
+        assert!(false);
+    }
+}
+
+#[test]
+#[cfg(windows)]
+fn test_process_environ() {
+    let p = std::process::Command::new("timeout.exe")
+        .arg("/t")
+        .arg("10")
+        .env("FOO", "BAR")
+        .env("OTHER", "VALUE")
+        .spawn()
+        .unwrap();
+    let pid = p.id() as sysinfo::Pid;
+
+    let mut s = sysinfo::System::new();
+    s.refresh_processes();
+
+    let processes = s.processes();
+    let p = processes.get(&pid);
+
+    if let Some(p) = p {
+        assert_eq!(p.pid(), pid);
+        assert!(p.environ().iter().any(|e| e == "FOO=BAR"));
+        assert!(p.environ().iter().any(|e| e == "OTHER=VALUE"));
+    } else {
+        assert!(false);
```

Comment:
Might be worth using `unix_like_cmd` like in `test_get_cmd_line`, don't you think?